apps/communex-roadmap/src/app/page.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import React from "react";
+
+import Stepper from "./components/Stepper";
+
+const steps = {
+  "Q3 2024": [
+    {
+      title: "Step 1",
+      description: "Step 1 description",
```

Comment:
Can we have this in a clean separate file as the actual way to input data in the roadmap page?